.github/workflows/acceptance-tests.yml

```diff
@@ -61,8 +67,8 @@ jobs:
           role-duration-seconds: 21600 # 6h
       - run: |
           echo "Testing with Go ${{ needs.get-go-version.outputs.go-version }}"
-          PACKER_ACC=1 go test \
-            -timeout 5h \
+          go test \
+            -timeout 2h \
```

Comment:
While I would assume they don't need 5 hours to run, I think it might be premature to lower that to 2h, it's a timeout, we can be large here, especially since if we're too optimistic this will `kill -9` the tests, and not clean-up the resources.
All that to say, I think we shouldn't change that now, I think it doesn't hurt to be pessimistic with the timeout here.